### PR TITLE
Some review feedback and other misc tweaks

### PR DESCRIPTION
**Pull request by alexcrichton**

### CODE REVIEW

File: .azure-pipelines/steps/install-clang.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+#
+# Original downloaded here came from
+# http://releases.llvm.org/7.0.0/LLVM-7.0.0-win64.exe
+- script: |
+    powershell -Command "iwr -outf %TEMP%\LLVM-7.0.0-win64.exe https://s3-us-west-1.amazonaws.com/rust-lang-ci2/rust-ci-mirror/LLVM-7.0.0-win64.exe"
+    set CLANG_DIR=%CD%\citools\clang-rust
+    %TEMP%\LLVM-7.0.0-win64.exe /S /NCRC /D=%CLANG_DIR%
+    set RUST_CONFIGURE_ARGS=%RUST_CONFIGURE_ARGS% --set llvm.clang-cl=%CLANG_DIR%\bin\clang-cl.exe
+    echo ##vso[task.setvariable variable=RUST_CONFIGURE_ARGS]%RUST_CONFIGURE_ARGS%
+  condition: and(eq(variables['Agent.OS'], 'Darwin'), eq(variables['MINGW_URL'],''))
```
Comment (pietroalbini) [inline .azure-pipelines/steps/install-clang.yml]: Pretty sure the agent OS is wrong here.

**Comment by pietroalbini** [comment]
Oh just noticed you're missing `stamp`.

**Comment by alexcrichton** [comment]
Ah that was intentionally removed in https://github.com/alexcrichton/rust/commit/79d28c203fbf46739d2e3f1f2f3e62bdf1ac5563. Build logs from azure have more timestamps than Travis, so we don't need `stamp` like we don't need it on AppVeyor since they've got timestamps on logs already too

**Comment by pietroalbini** [comment]
Where are you seeing the timestamps in the log lines? I opened one of your in-progress builds and there are none of them.

**Comment by alexcrichton** [comment]
Oh sorry yeah they don't show up in the web UI, but if you download the full log locally then it'll have timestamps on each line

**Pull request merged**